.github/dependabot.yml

```diff
@@ -14,3 +14,15 @@ updates:
       interval: "daily"
     commit-message:
       prefix: "chore"
+
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "chore"
+    ignore:
+      - dependency-name: "*"
```

Comment:
Won't this effectively disable dependabot for go, as we're ignoring all dependencies?

Reply:
Wait, now I see. This makes sense since it means dependabot only triggers for patch updates.